tools/mo/unit_tests/mo/frontend_ngraph_test.py

```diff
@@ -33,7 +33,7 @@ def test_frontends():
     status = subprocess.run(args, env=os.environ)
     assert not status.returncode
 
-
+@pytest.mark.skip(reason="Ticket - 113358")
```

Comment:
What is the exact problem with it? Let us discuss offline before skipping.

Reply:
@pavel-esir, did you have a chance to look at this? 
@akashchi, did you build moc_frontend for these tests and register a path to it in `LD_LIBRARY`?

Reply:
@pavel-esir highlighted that these tests are skipped in Azure due to the absence of the mock_api:
![image](https://github.com/openvinotoolkit/openvino/assets/65596953/472f2162-cee0-42f8-9a31-ab42ef25a5de)

[Example pipeline](https://dev.azure.com/openvinoci/dldt/_build/results?buildId=590660&view=logs&j=982848f2-7f8d-5e57-730a-8df1e5f991ba&t=640d3dcd-a1dd-5e6b-6d22-d23b2eb859ae)

I am unsure why they are executed in the GH Actions pipeline as I was copying the build commands from the Azure pipeline.

Reply:
@rkazants could you please link me to the example command used for building `moc_frontend` and registering its path in `LD_LIBRARY`?

Reply:
I think @akashchi was right that skipped them. @rkazants i think they can be definitely skipped at the moment. They cannot be patched quickly, they need to be rewritten.

>pavel-esir, did you have a chance to look at this?

yes, i did look at them. And in Andrey's environment openvino_mock_mo_frontend was successfully build and placed correctly inside `bin/intel64/Release/python`(without version) and for this very reason they were not skipped. 

But in the Azure they are placed wrongly somewhere else and since `openvino_mock_mo_frontend` was not found they are silently skipped:

https://github.com/openvinotoolkit/openvino/blob/5d399faa642cfafa6ee995bf689ad2dfb79bd7e3/tools/mo/unit_tests/mo/moc_frontend/moc_extractor_test_actual.py#L14-L26


If i build all `pybind_mock_frontend`, `openvino_mock_mo_frontend `, `mock_mo_python_api` et all. and look at the messages of the exact failed tests `test_frontends` (which calls `frontend_ngraph_test_actual`) and `test_moc_extractor` (which calls `moc_extractor_test_actual`) they are very outdated. Eg. they are called with 3 arguments including `fem`
https://github.com/openvinotoolkit/openvino/blob/0c9abf43a91fbe2dc95974b6cb2651bd883d1b12/tools/mo/unit_tests/mo/frontend_ngraph_test_actual.py#L98
but the current tools.mo.main has only 2 input arguments https://github.com/openvinotoolkit/openvino/blob/5d399faa642cfafa6ee995bf689ad2dfb79bd7e3/tools/mo/openvino/tools/mo/main.py#L19

even if removed redundant `fem` then other error occurs, mock frontends changed their names
![image](https://github.com/openvinotoolkit/openvino/assets/5703039/fc268838-1a01-45d4-aee3-0a827f63d22a)

Since they actually have been implicitly skipped for a long time they have become a little bit stale. Therefore if we want to keep this tests we need some time to fix/rewrite them.